contrib/babelfishpg_common/sql/upgrades/spatial_types--3.3.0--3.4.0.sql

```diff
@@ -853,7 +853,7 @@ CREATE OR REPLACE FUNCTION sys.GEOGRAPHY(sys.bpchar)
 	DECLARE
 		geog sys.GEOGRAPHY;
 	BEGIN
-		geog := (SELECT sys.bpcharToGeography_helper($1));
+		geog := (SELECT sys.bpcharToGeography_helper($1, 4326));
```

Comment:
Have we checked that if 4326 is default SRID?

Reply:
Yes, we verified that for Geography default SRID is 4326 and for Geometry it is 0